src/NexusMods.DataModel/Loadouts/IngestSteps/BackupFile.cs

```diff
@@ -0,0 +1,30 @@
+using NexusMods.DataModel.Loadouts.ApplySteps;
+using NexusMods.Hashing.xxHash64;
+using NexusMods.Paths;
+
+namespace NexusMods.DataModel.Loadouts.IngestSteps;
+
+/// <summary>
+/// Marks a file as requiring backup.
+/// </summary>
+/// <remarks>
+///    Used when a mod replaces an existing [game] file and the file
+///    is not already backed up.
+/// </remarks>
+public record BackupFile : IIngestStep
```

Comment:
`IngestSteps.BackupFile` clashes with `ApplySteps.BackupFile`. These files are almost identical, with the only difference being the interfaces the record implements. Both records should not have the same XML documentation.